docs/api.md

```diff
@@ -1951,6 +2002,36 @@ You can fully delete a link by issuing `DELETE` to its resource. This will remov
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+
+## Related Datasets [/v1]
+
+_(introduced: version 2022.3)_
+
+A Form Submission can affect a Dataset by creating Entities in it. In future versions, a Submission will also be able to update Entities. To know which Dataset is affected by a Form's Submissions, use following endpoints.
+
+Note that it is not necessary that a Form will save to all Properties of a Dataset, so the endpoint also returns a `inForm` flag against each property which is true only if the Form affects that Property.
+
+### For a published form [GET /v1/projects/{projectId}/forms/{xmlFormId}/dataset-diff]
+
+To know the Datasets and Entity Properties affected by a published Form, use this endpoint.
+
++ Response 200 (application/json)
+    This is the standard response
+
+    + Attributes (array[Dataset Diff])
+
+### For a draft form [GET /v1/projects/{projectId}/forms/{xmlFormId}/draft/dataset-diff]
+
+To know the Datasets and Entity Properties affected by a Draft Form, use this endpoint.
+
+This endpoint also returns an `isNew` flag against each Dataset and each of its Properties. This flag is true only if the Dataset/Property is new and is going to be created by publishing the Draft Form.
```

Comment:
I want to reword things in this section/subsections and add more details.... but not sure how yet. Going to work on that after I post these comments.